convert-hf-to-gguf.py

```diff
@@ -23,7 +23,9 @@
 
 if 'NO_LOCAL_GGUF' not in os.environ:
     sys.path.insert(1, str(Path(__file__).parent / 'gguf-py'))
-import gguf
+import importlib
+gguf = importlib.import_module("gguf-py.gguf")
+# import gguf
```

Comment:
This is a temporary fix to load the updated `gguf` package during development, because importing from local path complains about the dash in the directory name. When this PR is merged (or when I get the greenlight on the `gguf` package part), I will push the changes to the `gguf` package and replace these lines with `import gguf` again.

Reply:
> This is a temporary fix to load the updated `gguf` package during development, because importing from local path complains about the dash in the directory name.

But I think the `NO_LOCAL_GGUF` path insertion thing should already do this, no? If you happen to have the `NO_LOCAL_GGUF` environment variable defined, maybe try to `unset NO_LOCAL_GGUF`?

> When this PR is merged (or when I get the greenlight on the gguf package part), I will push the changes to the gguf package and replace these lines with import gguf again.

The source of the `gguf` package ***is*** `gguf-py` in the `llama.cpp` repo, from what I understand. So you don't need to separately push changes to "the `gguf` package", since you're already modifying it.

What could be done however, to eventually update the package in pypi, would be to bump the version and tag it as described in <https://github.com/ggerganov/llama.cpp/tree/master/gguf-py#automatic-publishing-with-ci>. But this can also be done separately.

Reply:
Oh. I had meant pypi updating, but I didn't realize it was taken straight from the local `gguf-py` directory. Strange, I'll look into this when I get home.

Reply:
Confirmed that it does seem like a simple `import gguf` does the trick. Fixed in c6ae1d6.